frontend/containers/AuctionsContainer.vue

```diff
@@ -80,6 +82,7 @@ export default Vue.extend({
                 if (!newAuctionId) {
                     const network = this.$route.query.network;
                     this.$router.push({ query: { network } });
+                    this.updateSingleAuction(newAuctionId);
```

Comment:
According to the `if` statement on line 82, `newAuctionId` is falsy, but you still call `updateSingleAuction` with it?

Reply:
Oops! Oversaw this.